server/modules/nodeDef/manager/nodeDefManagerLayout.js

```diff
@@ -0,0 +1,145 @@
+import * as R from 'ramda'
+
```

Comment:
call this NodeDefLayoutManager

Reply:
can you add some comment to the methods below? It will make code and features more readable

Reply:
ok, sure